src/components/DataBox/DataBox.jsx

```diff
@@ -10,15 +10,19 @@ const DataBox = () => {
     <div className={`data-box data-box-${theme}`}>
       <div className="data-box__info">
         <p>Repos</p>
-        <p>{state.profile.public_repos}</p>
+        <p>{state.profile.publicReposCount}</p>
+      </div>
+      <div className="data-box__info">
+        <p>Gists</p>
+        <p>{state.profile.publicGistsCount}</p>
       </div>
       <div className="data-box__info">
         <p>Followers</p>
-        <p>{state.profile.followers}</p>
+        <p>{state.profile.followersCount}</p>
       </div>
       <div className="data-box__info">
         <p>Following</p>
-        <p>{state.profile.following}</p>
+        <p>{state.profile.followingsCount}</p>
```

Comment:
imo these `<p>` tags should be `<span>` because they are not actually paragraphs